server/routes/utils/adHelpers.ts

```diff
@@ -532,9 +532,16 @@ export const isRealTimeTask = (task: any) => {
   return get(task, 'task_type', '').includes(REALTIME_TASK_TYPE_PREFIX);
 };
 
-export const getLatestTasksForDetectorQuery = (detectorId: string) => {
+export const getLatestTaskForDetectorQuery = (
+  detectorId: string,
+  realtime: boolean
+) => {
+  const taskTypes = realtime ? REALTIME_TASK_TYPES : HISTORICAL_TASK_TYPES;
   return {
-    size: 3,
+    size: 1,
+    sort: {
+      execution_start_time: SORT_DIRECTION.DESC,
```

Comment:
Could you add your explanation why there can be multiple real time tasks and we need to sort as comments? https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/69#discussion_r694227657

Reply:
Sure, will add.

Reply:
Added in this [commit](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/69/commits/fa3b25eb3e9f8aca606fba02d191f9d5b609df28)